drivers/clock_control/clock_control_nrf.c

```diff
@@ -620,6 +620,26 @@ static void clock_event_handler(nrfx_clock_evt_type_t event)
 	}
 }
 
+static void hfclkaudio_init(void)
+{
+#if DT_NODE_HAS_PROP(DT_NODELABEL(clock), hfclkaudio_frequency)
+	const uint32_t frequency =
```

Comment:
need to use static const, otherwise it might not end up in RO data.

Reply:
@anangl, @nordic-krch - is it possible to resolve this last change request?